packages/runtime-common/realm.ts

```diff
@@ -291,6 +292,7 @@ export class Realm {
     this.#realmSecretSeed = realmSecretSeed;
     this.#getIndexHTML = getIndexHTML;
     this.#useTestingDomain = Boolean(opts?.useTestingDomain);
+    let loader = virtualNetwork.createLoader();
```

Comment:
The most important change is this - loader is no longer passed into a realm, making an end to shared loaders.